assets/src/skate.d.ts

```diff
@@ -1,11 +1,18 @@
+import { directive } from "@babel/types"
```

Comment:
What does this do? I don't see `directive` used in this file.

Reply:
I think that was an accidental insertion by the editor. Removed.